CONTRIBUTING.md

```diff
@@ -174,40 +157,32 @@ applicable.
 Feel free to contribute standalone pull requests to add new tests or improve
 existing tests as well.
 
-See [Unit testing](https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
-for information on writing tests in Godot's C++ codebase.
+See [Unit testing](https://docs.blaziumengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
+for information on writing tests in Blazium's C++ codebase.
 
-## Contributing to Godot translations
+## Contributing to Blazium translations
 
-You can contribute to Godot translations on [Hosted Weblate](https://hosted.weblate.org/projects/godot-engine/),
+You can contribute to Blazium translations on [Hosted Weblate](https://hosted.weblate.org/projects/blazium-engine/),
 an open source and web-based translation platform.
 
-Please refer to our [editor and documentation localization guidelines](https://docs.godotengine.org/en/latest/contributing/documentation/editor_and_docs_localization.html)
+Please refer to Blazium's [editor and documentation localization guidelines](https://docs.blaziumengine.org/en/latest/contributing/documentation/editor_and_docs_localization.html)
 for an overview of the translation resources and what they correspond to.
 
 ## Communicating with developers
 
-The Godot Engine community has [many communication
-channels](https://godotengine.org/community), some used more for user-level
-discussions and support, others more for development discussions.
+The Blazium Engine community is largely centralized on the [Blazium Discord](https://discord.gg/redot) at this time, feel free to drop by!
 
 To communicate with developers (e.g. to discuss a feature you want to implement
 or a bug you want to fix), the following channels can be used:
 
-- [Godot Contributors Chat](https://chat.godotengine.org): You will
-  find most core developers there, so it's the go-to platform for direct chat
-  about Godot Engine development. Browse the [Directory](https://chat.godotengine.org/directory/channels)
-  for an overview of public channels focusing on various engine areas which you
-  might be interested in.
-- [Bug tracker](https://github.com/godotengine/godot/issues): If there is an
+- [Blazium Discord](https://discord.gg/redot): General community chat and development can all be found here.
+- [Bug tracker](https://github.com/Blazium-Engine/blazium-engine/issues): If there is an
```

Comment:
The link needs to be changed from this: (https://github.com/Blazium-Engine/blazium-engine/issues) to this: (https://github.com/blazium-engine/blazium/issues)

Reply:
Thanks, working an update now!